torch/quantization/fx/convert.py

```diff
@@ -535,4 +537,5 @@ def load_arg_remove(a: Argument) -> Argument:
     model = QuantizedGraphModule(model, act_post_process_removed_graph, preserved_attributes)
     if not is_reference:
         model = fold_weight(model, node_name_to_scope)
+        model = lower_to_fbgemm(model)
```

Comment:
we call the lowering function here in convert_fx, so no change of api for users with this PR

Reply:
makes sense, thanks.  Should we add a TODO here to lower to the correct choice from (fbgemm, qnnpack), and also verify that all qconfigs are valid for the selected backend from (fbgemm, qnnpack)?